lib/suites/LinkedDataSignature.js

```diff
@@ -193,6 +201,8 @@ module.exports = class LinkedDataSignature {
   async verify(framed, options) {
```

Comment:
noting that this existing API does not conform to the preferred validation convention.